src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -184,11 +184,9 @@ export class DictionaryModal extends React.Component {
                       type="text"
                       id="preferred_source"
                       name="preffered_source"
-                      placeholder="CIEL"
                       onChange={this.onChange}
                       value={data.preffered_sources}
                     >
-                      <option value="" />
```

Comment:
Has this change anything to do with the ticket?

Reply:
Yeah, it has changed as it was initially empty so I removed it to give the preferred sources a placeholder